core/server/data/migration/fixtures/006/01-transform-dates-into-utc.js

```diff
@@ -0,0 +1,201 @@
+var config = require('../../../../config'),
+    models = require(config.paths.corePath + '/server/models'),
+    sequence = require(config.paths.corePath + '/server/utils/sequence'),
+    moment = require('moment'),
+    Promise = require('bluebird'),
+    messagePrefix = 'Transform into UTC Dates: ',
+    settingsKey = '006/01',
+    _private = {};
+
+_private.getTZOffset = function getTZOffset(date) {
+    return date.getTimezoneOffset();
+};
+
+_private.getTZOffsetMax = function () {
+    return Math.max(Math.abs(new Date('2015-07-01').getTimezoneOffset()), Math.abs(new Date('2015-01-01').getTimezoneOffset()));
+};
+
+_private.addOffset = function (date) {
+    if (_private.noOffset) {
+        return moment(date).toDate();
+    }
+
+    return moment(date).add(_private.getTZOffset(date), 'minutes').toDate();
+};
+
+module.exports = function transformDatesIntoUTC(options, logger) {
+    var ServerTimezoneOffset = _private.getTZOffsetMax(),
+        settingsMigrations = null;
+
+    options.migration = true;
+
+    return sequence([
+        function () {
+            if (ServerTimezoneOffset === 0) {
+                return Promise.reject(new Error('skip'));
+            }
+
+            // pg stores dates with offset, so it's enough to force timezone UTC in the db connection (see data/db/connection.js)
+            if (config.database.isPostgreSQL()) {
+                return Promise.reject(new Error('skip'));
+            }
+
+            // sqlite stores UTC timestamps, but we will normalize the format to YYYY-MM-DD HH:mm:ss
+            if (config.database.client === 'sqlite3') {
+                _private.noOffset = true;
+            }
+
```

Comment:
Having tested this migration with very, very large datasets, there is a big gap between the previous message: 

> Migrations: Updating fixtures to 006

and the first message output by this migration:


> Migrations: Transform into UTC Dates: Updated datetime fields for Posts!

My recommendation would be to add one extra message around here somewhere when we know we're about to proceed on to updating data:

> Migrations: Transforming dates to UTC: (could take a while)....

This is similar to https://github.com/TryGhost/Ghost/blob/84f387785a96db1f0b1590425137fd73e2e492f4/core/server/data/migration/fixtures/004/07-add-post-tag-order.js#L56-L56